lib/dash/mpd_utils.js

```diff
@@ -269,8 +269,12 @@ shaka.dash.MpdUtils.createTimeline = function(
 
     for (var j = 0; j <= repeat; ++j) {
       var endTime = startTime + d;
-      timeline.push(
-          {start: (startTime / timescale), end: (endTime / timescale)});
+      item = {
```

Comment:
Yes, the `var` part is important. Otherwise you'll be creating a global variable.